src/DevicePathTracer.h

```diff
@@ -172,13 +144,19 @@ class DevicePathTracer {
             unsigned int recursionDepth,
             dim3 threadBlockSize,
             HostScene& hostScene,
-            std::shared_ptr<Framebuffer> framebuffer) : 
+            std::shared_ptr<Framebuffer> framebuffer,
+            CameraParams& cameraParams) : 
```

Comment:
VERY NIT: In this case, I don't think `CameraParams` is the best name to describe camera _object_. Maybe we could rename this class to something else? `CameraObject` seems to verbose, `Camera` lacks more information about the class. Or just name this to `CameraConfig`?

Reply:
good idea